src/app/technical-record/technical-record.component.ts

```diff
@@ -104,6 +106,9 @@ export class TechnicalRecordComponent implements OnChanges, OnInit {
       { panel: 'Technical record history', isOpened: toggleState },
       { panel: 'Plates', isOpened: toggleState }
     ];
+
+    this.indexOfSectionToScrollTo =

```

Comment:
We could probably move that into an until function, this will likely be reused for test history, if it is case the it would be nice to do it I think perhaps.